src/NLog.Targets.MauiLog/MauiLogTarget.Windows.cs

```diff
@@ -0,0 +1,77 @@
+﻿using System;
+using System.Diagnostics;
+using Layout = NLog.Layouts.Layout;
+
+namespace NLog.Targets;
+
+/// <summary>
+/// Output target for System.Diagnostics.Debugger.Log with fallback to Console when no debugger
+/// </summary>
+[Target("MauiLog")]
+[NLog.Targets.MauiLogLinker.Preserve]
+public class MauiLogTarget : TargetWithLayoutHeaderAndFooter
```

Comment:
Possible that the new namespace-behavior can be reverted, so it will be git-rename, instead git-delete and git-rename.

Reply:
Curious why this should be renamed to `MauiLogTarget.Windows.cs` since this logic should be for `netstandard2.0` (default platfrom when no platform-specific-version).

Reply:
I don't know what you mean by "Possible that the new namespace-behavior can be reverted, so it will be git-rename, instead git-delete and git-rename."
Also MAUI has no netstandard targets. it's always a specific platform

Reply:
Not a MAUI-expert, but it seems that MAUI-Apps are able to consume libraries that contains NetStandard-target. Would have expected that when building the MAUI-App then it would choose the assemblies, that matches the target being compiled (where fallback is NetStandard when nothing else matches).

Reply:
Regarding git-rename, then I like to preserve git-history. Right now this pull-request deletes the old file, and creates new file, instead of just a file-rename. Git will automatically recognize it being a rename when not making too many file-changes together with the file-name.